tests/external_executor/external_executor.py

```diff
@@ -38,6 +44,43 @@ def wrap_tx(stub):
     stub.EndTx(KV.ResponseDescription())
 
 
+key_priv_pem, _ = infra.crypto.generate_ec_keypair("secp256r1")
+cert = infra.crypto.generate_cert(key_priv_pem)
```

Comment:
Can we avoid the globals here and instead generate a new key/cert for each `test_`?

Reply:
Removed globals. But all tests still use the same key/cert for easier testing.